modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator_{context}"]
+= Installing the AWS Load Balancer Operator
+
+You can install the AWS Load Balancer Operator using the Red Hat {product-title} Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+.Procedure
+
+. Click *Operators* → *OperatorHub* in the OpenShift Web Console.
+. Click *AWS Load Balancer Operator*.
```

Comment:
Currently the name is "aws-load-balancer-operator" instead of "AWS Load Balancer Operator".
@arjunrn are we going to rename this?

Reply:
@arjunrn Should we change the name to "aws-load-balancer-operator" here?

Reply:
No we shouldn't change it here. I've [changed](https://github.com/openshift/aws-load-balancer-operator/pull/71) the display name in the CSV instead.

Reply:
Ack!